Gemfile

```diff
@@ -47,6 +49,7 @@ group :development do
   gem 'listen',                '3.1.5'
   gem 'spring',                '2.0.2'
   gem 'spring-watcher-listen', '2.0.1'
+  gem "erb2haml"    
```

Comment:
You don't need this gem, https://github.com/indirect/haml-rails has already a rake task for this: `bundle exec rake haml:erb2haml`